src/modules/drawing/lib/ProfileChart.js

```diff
@@ -293,14 +298,14 @@ export default class ProfileChart {
                 .attr('x', this.width / 2)
                 .attr('y', this.height + this.options.margin.bottom - 5)
                 .style('text-anchor', 'middle')
-            this.group
+            this.svg
                 .select('text.profile-legend')
                 .transition()
                 .duration(transitionTime)
                 .attr('text-anchor', 'end')
```

Comment:
This can go. I think that was me adding that. But it's not necessary as it doesn't change.